content_service.go

```diff
@@ -81,6 +81,18 @@ func (c *Entry) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// EntryContent represents the content of an entry
+type EntryContent []byte
+
+func (e *EntryContent) UnmarshalJSON(b []byte) error {
+	if n := len(b); n >= 2 && b[0] == 34 && b[n-1] == 34 { // string
+		*e = b[1 : n-1]
```

Comment:
- Would you mind explaining why this special handling is required for strings?
- I'm also curious if we need to unescape the escaped characters such as `\n` and `\"`.

Reply:
In case of yaml return, it's a string return. If we don't handle string return and just set `*e = b`, then what user got from `string(EntryContent)` is a string which contains `"`.
`\n` is no need to escaped in this case.
Hope my short explanation above help.

Reply:
Why is `UnmarshallJSON` involved when dealing with YAML?

Reply:
There are 2 reasons:
1. whole response is json but we want to deal with content (string) of yaml and also json payload.
2. `UnmarshallJSON` involved as `a hook` to decode EntryContent. Previously, `Content` is too general `interface{}`, user have no way to get content inside when return JSON.
That's why I handle it for both case:
1. Json user: decode himself by returned `bytes` payload
2. Yaml user: got full string of yaml content

Reply:
Thanks for explanation. Then, could you:

- add a test case that makes sure that this change still allows a user to retrieve a multi-line plaintext non-JSON file that contains a backslash (`\`) in verbatim?
- build the CLI with this pull request and make sure that CLI can still retrive a multi-line plaintext non-JSON file that contains a backslash (`\`) in verbatim?

Reply:
For example, you could try to store and retrieve a `foo.txt` whose content is:

```
foo
b"r
b\z
```

Reply:
Yes I will. Sorry for missing your comment. Let me add yours case to unit test.

Reply:
Thank you for your test case. I have fixed it and commit @trustin